media/src/androidMain/AndroidManifest.xml

```diff
@@ -1,2 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest package="dev.icerock.moko.media" />
\ No newline at end of file
+<manifest package="dev.icerock.moko.media"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <application>
+        <provider
+            android:name=".MokoMediaFileProvider"
+            android:authorities="${applicationId}.moko.media.provider"
```

Comment:
it really will use consumer app application id? or it just use package of library?

Reply:
It will be same as `context.applicationContext.packageName` in the application because if not then IllegalArgumentException will thrown in `FileProvider.getUriForFile` (but it doesn't happen)

Reply:
And also the library doesn't have its own applicationId